### ISSUE TITLE
OS X wheel for Python 3.8 missing GEOS?

### ISSUE THREAD

**Issue by dfremont** [Toblerity/Shapely#788]
## Expected behavior and actual behavior.

Per the readme, running `pip install shapely` should install Shapely without needing to first install GEOS. However, with a fresh installation of Python 3.8, I get the following error:

```
OSError: Could not find library geos_c or load any of its variants ['/Library/Frameworks/GEOS.framework/Versions/Current/GEOS', '/opt/local/lib/libgeos_c.dylib']
```

If I first install GEOS (I did `brew install geos`), then Shapely installs fine.

## Steps to reproduce the problem.

Run `pip install shapely`.

## Operating system

macOS 10.14.6

**Comment by sgillies** [comment]
There are no wheels on PyPI for Python 3.8.

**Comment by aleksanb** [comment]
What do we need to do to get new wheels published for 3.8?
I need this as well.

**Comment by sgillies** [comment]
Manylinux1 and macosx wheels for Shapely 1.7a2 and Python 3.8 are now on PyPI: https://pypi.org/project/Shapely/1.7a2/#files. To get them use pip's `--pre` option: `pip install -U --pre shapely`.
